test/e2e/e2e_test.go

```diff
@@ -234,6 +235,7 @@ type configurableClusterOptions struct {
 	NodePoolReplicas           int
 	SSHKeyFile                 string
 	NetworkType                string
+	ServicePublishingStrategy  string
```

Comment:
Can you put this option up wiht the KubeVirt ones by giving it the `KubeVirt` prefix

Reply:
done.